entrypoint.sh

```diff
@@ -1,3 +1,6 @@
 echo -n "AWS for Fluent Bit Container Image Version "
 cat /AWS_FOR_FLUENT_BIT_VERSION
-exec /fluent-bit/bin/fluent-bit -e /fluent-bit/firehose.so -e /fluent-bit/cloudwatch.so -e /fluent-bit/kinesis.so -c /fluent-bit/etc/fluent-bit.conf
+
+AWS_FOR_FLUENT_BIT_CONFIGFILE=${AWS_FOR_FLUENT_BIT_CONFIGFILE:-'/fluent-bit/etc/fluent-bit.conf'}
```

Comment:
I think there should be an underscore between CONFIG and FILE to make it all consistent, so `AWS_FOR_FLUENT_BIT_CONFIG_FILE`

Reply:
I prefer it to be `FLUENT_BIT_CONFIG_FILE` as aws-for-fluent-bit doesn't have a different config.

Reply:
@hossain-rayhan  I agree ! , I'll do the change !